utils/run_test.sh

```diff
@@ -142,31 +142,32 @@ if [ -d "/mnt/daos" ]; then
         run_test src/common/tests/btree.sh dyn perf ukey -s 20000
         BTREE_SIZE=20000
 
-        COMP="UTEST_vos_nvme"
+        COMP="UTEST_vos_nvme_md_on_ssd_bio_wal_vos_tests"
         if run_test_filter "nvme"; then
             cat /proc/meminfo
             AIO_DEV=$(mktemp /tmp/aio_dev_XXXXX)
             # Setup daos_nvme.conf
             NVME_CONF="/mnt/daos/daos_nvme.conf"
             cp -f src/vos/tests/daos_nvme.conf ${NVME_CONF}
             sed -i "s+\"filename\": \".*\"+\"filename\": \"${AIO_DEV}\"+g" ${NVME_CONF}
+            export VOS_BDEV_CLASS="AIO"
         fi
+        COMP="UTEST_vos_nvme"
         if run_test_filter "sudo -E ${SL_PREFIX}/bin/vos_tests -a"; then
             cat /proc/meminfo
             # Setup AIO device
             dd if=/dev/zero of="${AIO_DEV}" bs=1G count=4
 
-            export VOS_BDEV_CLASS="AIO"
             run_test "sudo -E ${SL_PREFIX}/bin/vos_tests" -a
         fi
 
         COMP="UTEST_vos_nvme_md_on_ssd"
         if run_test_filter "sudo -E ${SL_PREFIX}/bin/vos_tests -a"; then
```

Comment:
Instead of argument "-a", shouldn't it be "-A 50"?

Reply:
My understanding is that is an original attribute check which requires 4G aio file. @jolivier23 Is that correct?

Reply:
Right, it should be "-A 50". The change is pushed.